packages/react-utilities/src/compose/types.ts

```diff
@@ -34,9 +34,10 @@ export type ShorthandProps<TProps extends ComponentProps = {}> =
         children?: TProps['children'] | ShorthandRenderFunction<TProps>;
       });
 
-export type ObjectShorthandProps<TProps extends ComponentProps = {}> = TProps & {
-  children?: TProps['children'] | ShorthandRenderFunction<TProps>;
-};
+export type ObjectShorthandProps<TProps extends ComponentProps = {}> = TProps &
+  Omit<ComponentProps, 'children'> & {
```

Comment:
Should we also update `ShorthandProps` type?

Reply:
My 2 cents, probably not since there is already a way to convert it via `ResolvedShorthandProps` below?

Or how did you envision updating `ShorthandProps`?

Reply:
I updated `ShorthandProps` to accept `ObjectShorthandProps` as a possible type, rather than duplicating the definition, which can (and did) get out of date.